config/routes.rb

```diff
@@ -19,6 +19,7 @@
       resources :service_plans, :only => [:index, :show] do
         resources :service_instances, :only => [:index]
       end
+      post "/service_plans/:id", :to => "service_plans#order"
```

Comment:
What does this create?  Is it a ServiceInstance?  If so, should we post to service_plans/:id/service_instances instead?

Reply:
@mkanoor @gmcculloug ^

Reply:
So this is still a question of if we should post to `service_plan/:id/#order` or `service_plan/:id/service_instances#create` right?  I don't feel strongly either way, I assume the latter is "more correct" from an API standpoint?

Reply:
cc @Fryguy Any thoughts on this?